test/unit/document/content/create-document-content.test.js

```diff
@@ -33,7 +33,7 @@ describe('generate document content', () => {
 
   test('includes Agreement details', () => {
     expect(result[1].stack[4].ul[0]).toBe(`Agreement number: ${mockData.reference} \n\n`)
-    expect(result[1].stack[4].ul[1]).toBe(`Agreement holder: ${mockData.orgName} - ${mockData.sbi} \n\n`)
+    expect(result[1].stack[4].ul[1]).toBe(`Agreement holder: ${mockData.name} - ${mockData.sbi} \n\n`)
```

Comment:
shall we add couple of test for unhappy path ,  ej when mockData.name is missing  and when data is { } (empty ) ,

it should send the email with missing data or no sending it at all ,however it should not fail